src/utils/logs.ts

```diff
@@ -312,6 +313,13 @@ export function logDeprecation(
 	};
 }
 
+export function logConstVariableReassignError() {
+	return {
+		code: CONST_REASSIGN,
+		message: 'Cannot reassign to a variable declared with `const`'
```

Comment:
```suggestion
		message: 'Cannot reassign a variable declared with `const`'
```

Just a minor wording change. I think it is "assign to something" but "reassign something" https://www.oxfordlearnersdictionaries.com/definition/english/reassign